ts-packages/web/src/app/(social)/my-follower/page.client.tsx

```
@@ -203,7 +209,7 @@ function FollowingInfo({
 
               <div
                 id="user-profile-description"
-                className="font-medium text-[12px] text-neutral-300"
+                className="font-medium text-[12px] text-neutral-300 light:text-neutral-400"
                 dangerouslySetInnerHTML={{ __html: user.html_contents }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)